src/js/actions/DialogActions.js

```diff
@@ -1,72 +1,84 @@
+var Util = require("../helpers/Util");
 var AppDispatcher = require("../AppDispatcher");
 var DialogEvents = require("../events/DialogEvents");
+var DialogTypes = require("../constants/DialogTypes");
+var dialogScheme = require("../stores/schemes/dialogScheme");
 
 var DialogActions = {
-  alert: function (message, dismissButtonLabel = "OK") {
-    const dialogId = Symbol(message);
-    AppDispatcher.dispatchNext({
-      actionType: DialogEvents.ALERT_SHOW,
-      dialogId: dialogId,
-      dismissButtonLabel: dismissButtonLabel,
-      message: message
-    });
-    return dialogId;
-  },
-  alertDismiss: function (dialogId) {
-    AppDispatcher.dispatchNext({
-      actionType: DialogEvents.ALERT_DISMISS,
-      dialogId: dialogId
-    });
-  },
-  confirm: function (message, successButtonLabel = "OK") {
-    const dialogId = Symbol(message);
+  dialog: function (data) {
```

Comment:
@aldipower Ok, then... As discussed, let's only expose those three actions (`alert`, `confirm`, `prompt`) and refactor the rest of the whole action, store and component dialog stuff to simplify things even further and get rid of the `showDialog`, `acceptDialog`, `dismissDialog` actions.

Proposal (DialogActions):
* alert(dialogScheme):Promise
* prompt(dialogScheme):Promise
* confirm(dialogScheme):Promise

What do you think?